python/django_bolt/serialization.py

```
@@ -422,15 +422,18 @@ def _compile_serializer_values_evaluators(serializer_cls: Any) -> tuple[Any, Any
     model_ok: dict[type, bool] = {}
 
     def evaluate_sync(queryset: QuerySet) -> list[Any]:
+        queryset = serializer_cls._prepare_queryset(queryset)  # Config.annotations & co.
         ok = model_ok.get(queryset.model)
         if ok is False:
-            return list(queryset)
+            # Project here, on the ORM thread, so model properties may still
+            # query (an N+1, never SynchronousOnlyOperation on the loop).
+            return _dump_many_sync(serializer_cls, list(queryset), False)
```

Ruff (0.16.1)

[warning] Boolean positional value in function call

(FBT003)